### PR TITLE
UI Configuration via Tenant

### PR DESCRIPTION
**Pull request by andreitr**
TLDR; We can now define Tenant UI elements via the Tenant UI configuration.

Untitled (4)

## QA 

- [x]  Proposal create (andrei)
- [x]  Proposal vote (andrei)
- [x]  Proposal delete (andrei)
- [x]  Admin Governor settings (andrei)
- [x]  Proposal type settings (andrei)
- [x] Delegate statement update (andrei)
- [x] Delegate votes 
- [x] Un delegate votes

---

## PR-Codex overview
This PR introduces support for `Ether.fi` alongside `Optimism` and `ENS` namespaces. It also updates references to `Tenant.getInstance()` to `Tenant.current()` for consistency.

### Detailed summary
- Added support for `Ether.fi` namespace
- Updated references from `Tenant.getInstance()` to `Tenant.current()` for consistency

> The following files were skipped due to too many changes: `src/app/api/common/metrics/getMetrics.ts`, `src/lib/tenant/tenantSlugFactory.ts`, `src/lib/alligatorUtils.ts`, `src/components/Proposals/ProposalsFilter/ProposalsFilter.tsx`, `src/app/admin/page.tsx`, `src/app/retropgf/3/application/[id]/page.tsx`, `src/app/retropgf/3/page.tsx`, `src/lib/voteUtils.ts`, `src/components/Proposals/ProposalPage/CastVoteDialog/CastVoteDialog.tsx`, `src/assets/tenant/agora.svg`, `src/app/api/common/proposals/getNeedsMyVoteProposals.ts`, `src/app/proposals/create/page.jsx`, `src/app/delegates/page.jsx`, `src/app/api/common/quorum/getQuorum.ts`, `src/lib/tokenUtils.ts`, `src/app/retropgf/3/summary/page.tsx`, `src/components/Header/LogoLink.jsx`, `src/components/Delegates/DelegatesTabs/DelegatesTabs.tsx`, `src/lib/tenant/tenantUI.ts`, `src/components/Proposals/ProposalCreation/CastProposalDialog.tsx`, `src/lib/tenant/tenantContractFactory.ts`, `src/components/Header/Navbar.jsx`, `src/app/api/common/votes/getVotes.ts`, `src/components/Votes/ApprovalProposalVotesList/ApprovalProposalVotesList.tsx`, `src/lib/tenant/tenant.ts`, `src/app/api/common/proposals/getProposals.ts`, `src/components/Hero/Hero.jsx`, `src/app/api/common/voting-power/getVotingPower.ts`, `src/app/page.jsx`, `prisma/schema.prisma`, `src/assets/tenant/optimism_logo.svg`, `src/app/delegates/[addressOrENSName]/page.tsx`, `src/app/api/common/delegates/getDelegates.ts`, `src/assets/tenant/ens_logo.svg`, `src/assets/tenant/etherfi_logo.svg`, `src/app/api/common/delegations/getDelegations.ts`, `src/lib/tenant/tenantUIFactory.ts`, `src/assets/tenant/etherfi_hero.svg`, `src/assets/tenant/optimism_hero.svg`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/app/api/common/delegates/getDelegates.ts [hunk 1]
```diff
@@ -61,7 +62,7 @@ export async function getDelegates({
             take
           );
         default:
-          return prisma[`${namespace}Delegates`].findMany({
+          return (prisma as any)[`${namespace}Delegates`].findMany({
```
Comment (stepandel) [inline src/app/api/common/delegates/getDelegates.ts:65]: don't love this but don't have a better suggesting atm

I think the typechecks are generally helpful. 

One option could be creating ghost views in Prostgres / prisma

File: prisma/schema.prisma [hunk 2]
```diff
@@ -610,6 +610,7 @@ enum DaoSlug {
   OP
   ENS
   UNI
+  ETHFI
   NOUNS
   LYRA
   ETHERFI
```
Comment (stepandel) [inline prisma/schema.prisma:616]: This `ETHERFI` slug reflects the DB state. I can update to `ETHFI` on the DB if you think it's a better name

Comment (andreitr) [reply prisma/schema.prisma:616]: Good catch, I will remove that. Their token is ETHFI - i think ETHERFI makes more sense as a namespace.

File: src/app/api/common/delegates/getDelegates.ts (hunk 1 above)
Comment (andreitr) [reply src/app/api/common/delegates/getDelegates.ts:65]: @stepandel same. I briefly looked into ways of abstracting this further but there wasn't an easy solution. I like the idea of a generic ghost view though.

**Review by stepandel — APPROVED** [review]
LGTM!

**Pull request merged**